packages/fxa-shared/l10n/branding.ftl

```diff
@@ -25,7 +25,12 @@
 -product-mozilla-account = Mozilla account
 
 # "accounts" can and should be localized, "Mozilla" must be treated as a brand. Plural "Mozilla accounts" is used when referring to something affecting all Mozilla accounts, not just the individual's account.
--product-mozilla-accounts = Mozilla accounts
+# We almost always want "accounts" lowercased, however, there is at least one special case where we consider capitalized "Accounts" as headline casing of a legal document or reference to a legal document.
```

Comment:
@vpomerleau @bcolsson Given the recent decision about capitalizing "A" also for the email "sender", which will be done in a follow up, maybe we should update the comment in case that may change a translation for the uppercase version? Or should we update the comment then, or leave as-is?

Reply:
```suggestion
# "accounts" should be lowercase in almost all cases. Uppercase is reserved for special use cases where headline case is necessary, for example legal document names and references.
```
We could adjust the comment, here's an example. I think we'd want to rely on comments explaining why capitalized "accounts" is used for the strings we are using the `uppercase` parameter.